src/store.js

```diff
@@ -1,15 +1,18 @@
 import Vue from 'vue';
 import Vuex from 'vuex';
 import VueApollo from 'vue-apollo';
-import ApolloClient from 'apollo-boost';
+import { ApolloClient, HttpLink } from '@apollo/client/core';
+import { InMemoryCache } from '@apollo/client/cache';
 
 import createStore from '@scaife-viewer/store';
 
-const client = new ApolloClient({
-  uri:
-    process.env.VUE_APP_ATLAS_GRAPHQL_ENDPOINT ||
+const link = new HttpLink({
+  uri: process.env.VUE_APP_ATLAS_GRAPHQL_ENDPOINT ||
     'https://explorehomer-atlas-dev.scaife-viewer.org/graphql/',
 });
+const cache = new InMemoryCache();
```

Comment:
@paltman I looked through the docs a bit about the error you mentioned:

https://www.apollographql.com/docs/react/caching/cache-field-behavior/#merging-non-normalized-objects

> Alternatively, if you prefer to keep the default behavior of completely replacing the existing data with the incoming data, while also silencing the warnings, the following merge function will explicitly permit replacement:
> Since writing this kind of merge function can become repetitive, the following shorthand will provide the same behavior:

I tested the change below and I think this will indeed silence the error.
```suggestion
const cache = new InMemoryCache({
  typePolicies: {
    Query: {
      fields: {
        passageTextParts: {
          merge: false,
        },
      },
    },
  },
});
```

I still think there is a pass for both the frontend and backend we can make to improve how "cacheable" data is, but getting us onto the latest Apollo Client is the first step.